content/podcast/2023/ep10_flux_transcripts_errors.md

```diff
@@ -0,0 +1,44 @@
+title=The Flux executor, bytesize transcripts and improved error messaging
+episode=10
+description=The Flux executor, bytesize transcripts and improved error messaging.
+date=2023-01-18
+type=podcast
+subtype=News and Views
+youtubeid=5UDIGQVo9hA
+image=img/podcast_ep10.jpg
+tags=news and views,opensource,community
+status=published
+author= Developer advocates
+icon=logo_podcast_channels.jpg
+~~~~~~
+
+In this News and Views episode, [Chris Hakkaart](https://twitter.com/chris_hakk) and [Marcel Ribeiro-Dantas](https://twitter.com/mribeirodantas) discuss the hottest topics in the Nextflow world.
+
+<!-- end-archive-description -->
+
+#### The Flux executor
+
+- [Flux](https://flux-framework.org/) is a job manager akin to Slurm, Sun Grid Engine, or similar and has been added to Nextflow as an executor.
```

Comment:
```suggestion
- [Flux](https://flux-framework.org/) is a job scheduler akin to Slurm, Sun Grid Engine, or similar but with some extra features and has been added to Nextflow as an executor.
```

Reply:
I know Vanessa mentioned job manager in the PR but I suppose it's due to the fact they call flux a _resource manager_ (based on the extra bits it does), but when it comes to what Vanessa is referring to at this part, I think job scheduler is more appropriate.